### PR TITLE
Add Friends

### PR DESCRIPTION
**Pull request by TheJokersThief**

### CODE REVIEW

File: twitter/friends.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+// FriendIDParams are the parameters for FriendService.Ids
+type FriendIDParams struct {
+	UserID     int64  `url:"user_id,omitempty"`
+	ScreenName string `url:"screen_name,omitempty"`
+	Cursor     int64  `url:"cursor,omitempty"`
+	Count      int    `url:"count,omitempty"`
+}
+
+// IDs returns a cursored collection of user ids following the specified user.
```
Comment (dghubble) [inline twitter/friends.go]: copy/paste: user ids for every user the specified user is following (and wrap at 80 char plz)

File: twitter/friends.go [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+type FriendListParams struct {
+	UserID              int64  `url:"user_id,omitempty"`
+	ScreenName          string `url:"screen_name,omitempty"`
+	Cursor              int    `url:"cursor,omitempty"`
+	Count               int    `url:"count,omitempty"`
+	SkipStatus          *bool  `url:"skip_status,omitempty"`
+	IncludeUserEntities *bool  `url:"include_user_entities,omitempty"`
+}
+
+// List returns a cursored collection of Users following the specified user.
```
Comment (dghubble) [inline twitter/friends.go]: ...collection of Users the specified user is following.

File: twitter/friends.go [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+}
+
+// newFriendService returns a new FriendService.
+func newFriendService(sling *sling.Sling) *FriendService {
+	return &FriendService{
+		sling: sling.Path("friends/"),
+	}
+}
+
+// FriendIDParams are the parameters for FriendService.Ids
```
Comment (dghubble) [inline twitter/friends.go]: .Ids -> .IDs

**Comment by dghubble** [comment]
Thanks! Can you also add `Friends (following)` to the API list in the README?

File: twitter/friends.go (hunk 1 above)
Comment (shurcooL) [inline twitter/friends.go]: Why wrap at 80 characters exactly?

IMO, [this change](https://github.com/dghubble/go-twitter/pull/24/commits/de92f98de5f6f4a2e9e3cb05752c555863516087) hurts readability, it does not improve it.

See https://github.com/golang/go/issues/11639#issuecomment-120195067.

Of course it's your project so you're welcome to dictate it's style, but I'd like to understand what's your rationale.

Comment (dghubble) [inline twitter/friends.go]: The main point is that the comment is wrong. Those the specified user is following vs user ids following the user.

Yes, please break comment lines so they don't exceed 80 characters for consistency with this project and convenience with other tools.

File: twitter/friends.go (hunk 2 above)
Comment (dghubble) [inline twitter/friends.go]: same ^

**Comment by dghubble** [comment]
Still one function with the wrong documentation line. Also, I should have clarified, code does not need to be wrapped at 80, just any long comments.

**Comment by dghubble** [comment]
Hm, closed? I do appreciate these contributions as I don't have as much time to spend on each individual project anymore. Anyway, thanks for the effort. I can get to adding additional endpoints like these and verifying them when I have some time.

**Pull request closed without merging**